CMakeLists.txt

```diff
@@ -27,7 +27,15 @@ target_include_directories(oboe
                            PRIVATE src 
                            PUBLIC include)
 
+ # Compile Flags:
+ #     Enable -Werror when building debug config
+ #     Enable -Ofast
 target_compile_options(oboe PRIVATE -std=c++11
                             PRIVATE -Wall
-                            PRIVATE "$<$<CONFIG:DEBUG>:-Werror>") # Only include -Werror when building debug config
+                            PRIVATE "$<$<CONFIG:DEBUG>:-Werror>"
+                            PRIVATE -Ofast)
+
+# Enable logging when debugging
+# target_compile_options(oboe PUBLIC -DOBOE_ENABLE_LOGGING=1)
```

Comment:
Set `OBOE_ENABLE_LOGGING=1` for debug configuration and not for release

Reply:
restore its original for this one.